### ISSUE TITLE
Thoughts to a license

### ISSUE THREAD

**Issue by coburnw** [Tengo10/pinout-overview#10]
Have you given any thought to, or have any interest in assigning a license to your project?

Regards,
c.

**Comment by vazhnov** [comment]
Could you please add information about the license of this repository — is it allowed to copy, change and use the code and images in any purpose?
It would be great to see any popular open source license.

**Comment by Tengo10** [comment]
yes, I'm gonna look into it. I've been quite busy the past few months unfortunately. I have no clue how any of this really works but I imagine the image would be on a difference license than the code. Maybe CC BY-(SA) for the SVGs and MIT for the code.

**Comment by vazhnov** [comment]
MIT license for the code and CC BY (or CC BY-SA) for images — sounds awesome!
